<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmHostInfo</class>
 <widget class="QWidget" name="FrmHostInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="frmHostInfo">
     <property name="minimumSize">
      <size>
       <width>380</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>380</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stckForm">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageData">
         <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,1,100">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="lblHostName">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Host Name</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblTaskAgentMode">
                <property name="text">
                 <string>CONTAINER</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="lblLoadAvgs">
                <property name="text">
                 <string>Load Avg: #.## / #.## / #.##</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblCPULoad">
                <property name="text">
                 <string>Overall CPU load: ##.##% over %d s</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stckCharts">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="pageLoad">
             <layout class="QVBoxLayout" name="verticalLayout_11" stretch="1,1000">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Avg.Host Load</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSimpleTickerGraph" name="wdgChartLoad" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>70</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="pageCPU">
             <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1000">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Overall CPU load</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSimpleTickerGraph" name="wdgChartCPU" native="true"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="pageTasks">
             <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,1000">
              <property name="spacing">
               <number>2</number>
              </property>
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <pointsize>8</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Num of Total Tasks</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSimpleTickerGraph" name="wdgChartTasks" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageInfo">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="lblInfo">
            <property name="text">
             <string>Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz 
x86_64 (GenuineIntel)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QToolButton" name="tbtnLoad">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtnCPU">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CPU</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtnTasks">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Tasks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtnInfo">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Info</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollTasksInfo">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>68</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSimpleTickerGraph</class>
   <extends>QWidget</extends>
   <header>qsimpletickergraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>tbtnLoad</sender>
   <signal>clicked()</signal>
   <receiver>FrmHostInfo</receiver>
   <slot>showLoadPanel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbtnCPU</sender>
   <signal>clicked()</signal>
   <receiver>FrmHostInfo</receiver>
   <slot>showCPUPanel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbtnTasks</sender>
   <signal>clicked()</signal>
   <receiver>FrmHostInfo</receiver>
   <slot>showTasksPanel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbtnInfo</sender>
   <signal>clicked()</signal>
   <receiver>FrmHostInfo</receiver>
   <slot>showInfoPanel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showLoadPanel()</slot>
  <slot>showCPUPanel()</slot>
  <slot>showTasksPanel()</slot>
  <slot>showInfoPanel()</slot>
 </slots>
</ui>
